test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityTests.cs

```diff
@@ -9,6 +9,58 @@ namespace NuGet.Test
     public class CompatibilityTests
     {
         [Theory]
+        // net5.0 and later is not compat with net1.0 through net 4.x.xxx.
+        // users must use those TFMs via AssetTargetFallback, which will be set 
+        [InlineData("net5.0", "net452", false)]
+        [InlineData("net5.0", "net462", false)]
+        [InlineData("net5.0", "net472", false)]
+        [InlineData("net5.0", "net482", false)]
+        [InlineData("net50", "net5.0", true)]
+        [InlineData("net5.0", "net50", true)]
+        
+        // netcoreapp 5.0 and later are still valid for now
+        [InlineData("netcoreapp5.0", "netcoreapp5.0", true)]
+        [InlineData("net5.0", "netcoreapp5.0", true)]
+        [InlineData("netcoreapp6.0", "netcoreapp5.0", true)]
+
+        // net5.0 is compat with netstandard 1 through 2.1
+        [InlineData("net5.0", "netstandard2.2", false)]
+        [InlineData("net5.0", "netstandard2.1", true)]
+        [InlineData("net5.0", "netstandard2.0", true)]
+        [InlineData("net5.0", "netstandard1.7", true)]
+        [InlineData("net5.0", "netstandard1.6", true)]
+        [InlineData("net5.0", "netstandard1.5", true)]
+        [InlineData("net5.0", "netstandard1.4", true)]
+        [InlineData("net5.0", "netstandard1.3", true)]
+        [InlineData("net5.0", "netstandard1.2", true)]
+        [InlineData("net5.0", "netstandard1.1", true)]
+        [InlineData("net5.0", "netstandard1.0", true)]
+
+        // net5.0 and later is compat with netcoreapp3.1
+        [InlineData("net4.9", "netcoreapp3.1", false)]
+        [InlineData("net5.0", "netcoreapp3.1", true)]
+        [InlineData("net5.1", "netcoreapp3.1", true)]
+        [InlineData("net6.0", "netcoreapp3.1", true)]
+
+        // net5.0-<osname> is not yet supported with this change.
```

Comment:
```suggestion
        // net5.0-<osname> is not supported yet
```

Not need to have the with this change part.